docs/contributing-integrations.rst

```diff
@@ -179,6 +174,78 @@ to start the APM test agent, and re-run the test. Use `--pass-env` as described
 test agent. Once the run finishes, the snapshot file will have been regenerated.
 
 
+Writing Integration Tests for Your Integration
+++++++++++++++++++++++++++++++++++++++++++++++
+
+These instructions describe the general approach of writing new integration tests for a library integration.
+They use the Flask integration tests as a teaching example. Referencing these instructions against
+``tests/contrib/flask/test_flask_snapshot.py`` and ``tests/contrib/flask/app.py`` may be helpful.
+
+1. Make sure a directory for your integration exists under ``tests/contrib``
+2. Create a new file ``tests/contrib/<integration>/test_<integration>_snapshot.py``
+3. Make sure a ``Venv`` instance exists in ``Riotfile.py`` that references your ``contrib`` subdirectory.
+   Create one if it doesn't exist. Note the name of this ``Venv`` - this is the "test suite name".
```

Comment:
should we provide a copy/paste-able example?

Reply:
Sure, I'll add that in a future change